test/powershell/engine/Api/InitialSessionState.Tests.ps1

```diff
@@ -115,16 +115,7 @@ Describe "TypeTable duplicate types in reused runspace InitialSessionState TypeT
         It "Verifies that shared TypeTable is not allowed in ISS" {
 
             # Process TypeTable types from ISS.
-            $errorId = ""
-            try
-            {
-                $rs.Open()
-                throw "No Exception!"
-            }
-            catch
-            {
-                $_.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -Be "ErrorsUpdatingTypes"
-            }
+            { $rs.Open() } | Should -Throw -ErrorId "RuntimeException"
```

Comment:
Since the original code was checking the InnerException, you should do something like:

```powershell
$e = { $rs.Open() } | Should -Throw -ErrorId "RuntimeException" -PassThru
$e.Exception.InnerException.ErrorRecord.FullyQualifiedErrorId | Should -BeExactly "ErrorsUpdatingTypes"
```